pkg/server/config/config.go

```diff
@@ -110,6 +110,29 @@ func GetWorkspaceLogFilePath(workspaceId string) (string, error) {
 	return filePath, nil
 }
 
+func DeleteWorkspaceLogs(workspaceId string) error {
+	projectLogsDir, err := GetWorkspaceLogsDir()
+	if err != nil {
+		return err
+	}
+
+	workspaceDir := path.Join(projectLogsDir, workspaceId)
+
+	_, err = os.Stat(workspaceDir)
+	if os.IsNotExist(err) {
+		return nil
+	} else if err != nil {
+		return err
+	}
+
+	err = os.RemoveAll(workspaceDir)
+	if err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
Not a maintainer, but it looks like this snippet can be simplified to the following:

```suggestion
	return os.RemoveAll(workspaceDir)
```

Reply:
@kemzeb you're definitely right. Nice catch!

@vineeth-vk11 please apply the changes above.